lingua_franca/lang/parse_es.py

```diff
@@ -120,7 +120,7 @@ def extractnumber_es(text, short_scale=True, ordinals=False):
             break
 
         # number word and fraction
-        ands = ["e"]
+        ands = ["e", "y"]
```

Comment:
"e" as number or fractional separator doesn't exist in spanish. That's copy pasted from PT or IT. 
"e" can be used as a "y" only and if only the next word begins with an "i", to avoid cacophonies.

Reply:
So it can't be used with any numbers?

Reply:
there is no number starting with "i", so no.

Reply:
Thanks I'll remove the "e"

Reply:
Just for clarity: the test phrace "catorce e milésima" should be "catorce y milésima"

Reply:
TL;DR: yes, but for that example in particular, I also commented this https://github.com/MycroftAI/lingua-franca/pull/62#discussion_r360832559.

But yes, you're right, "**and**" in spanish is "**y**". Always... on numbering. 
It is always "y" when naming numbers and fractions. "Y" is _always_ placed between tens and units. You have some exceptions, like numbers from 11 to 29

45: cuarenta y cinco 
1492: mil cuatrocientos noventa y dos
2.856.729: dos millones ochocientos cincuenta y seis mil setecientos veintinueve
11: once
25: veinticinco

You can find some exceptions finding an "e" when enumerating and the forthcoming word begins with an "i", for example: "Eres guapo e inteligente" (You are handsome and intelligent), as the phonema of "y" and "i" is the same, so we put an "e" to avoid the cacophony.

Returning to numbers, as no number begins with an "i" (uno, dos tres, cuatro, cinco, seis, siete, ocho, nueve, diez, cien, mil, millon, etc), you will never ever see an "e".

Reply:
Thanks for the explanation! :+1: